src/gui/screen_menu_move.cpp

```diff
@@ -64,35 +66,62 @@ class MI_AXIS_E : public MI_AXIS<3, 5, 8> {
         SetVal(0);              // Reset spin before change
         lastQueuedPos = 0;      // zero it out so we wont go back when we exit the spinner
     }
-    void CheckNozzleTemp() {
-        bool temp_ok = (marlin_vars()->target_nozzle > MenuVars::GetExtrudeMinTemp());
-        if (temp_ok) {
-            if (!IsEnabled()) {
-                Enable();
-            }
-        } else {
-            if (IsEnabled())
-                Disable();
-        }
+};
+
+class DUMMY_AXIS_E : public WI_FORMATABLE_LABEL_t<int> {
+    virtual void click(IWindowMenu &window_menu) override {
+        PreheatStatus::Dialog(PreheatMode::None, RetAndCool_t::Both);
     }
+
+public:
+    DUMMY_AXIS_E()
+        : WI_FORMATABLE_LABEL_t<int>(_(MenuVars::labels[MARLIN_VAR_INDEX_E]), 0, is_enabled_t::yes, is_hidden_t::no, 0, [&](char *buffer) {
+            if (marlin_vars()->target_nozzle < MenuVars::GetExtrudeMinTemp()) {
+                snprintf(buffer, GuiDefaults::infoMaxLen, "Low temp");
+            } else {
+                snprintf(buffer, GuiDefaults::infoMaxLen, "Heating");
+            }
+        }) {}
 };
 
 using MI_AXIS_X = MI_AXIS<0, 5, 8>;
 using MI_AXIS_Y = MI_AXIS<1, 5, 8>;
 using MI_AXIS_Z = MI_AXIS<2, 1, 4>;
 
-using Screen = ScreenMenu<EFooter::On, MI_RETURN, MI_AXIS_X, MI_AXIS_Y, MI_AXIS_Z, MI_AXIS_E>;
+using Screen = ScreenMenu<EFooter::On, MI_RETURN, MI_AXIS_X, MI_AXIS_Y, MI_AXIS_Z, MI_AXIS_E, DUMMY_AXIS_E>;
 
 class ScreenMenuMove : public Screen {
     float prev_accel;
 
+    void CheckNozzleTemp() {
+        MI_AXIS_E *spinner = &Item<MI_AXIS_E>();
+        DUMMY_AXIS_E *dummy = &Item<DUMMY_AXIS_E>();
+        bool temp_ok = (marlin_vars()->temp_nozzle > MenuVars::GetExtrudeMinTemp());
+        if (temp_ok) {
+            if (spinner->IsHidden()) {
+                spinner->Show();
+            }
+            if (!dummy->IsHidden()) {
+                dummy->Hide();
+            }
+        } else {
+            if (!spinner->IsHidden()) {
+                spinner->Hide();
+            }
+            if (dummy->IsHidden()) {
+                dummy->Show();
+            }
+        }
```

Comment:
```suggestion
        if (temp_ok) {
            spinner->Show();
            dummy->Hide();
        } else {
            spinner->Hide();
            dummy->Show();
        }
```
If we had `SetVisibility(bool)` then we can have just:
```
spinner->SetVisibility(temp_ok);
dummy->SetVisibility(!temp_ok);
```
No `if` needed.

Reply:
Changed